ginga/rv/plugins/Catalogs.py

```diff
@@ -9,40 +9,33 @@
 channel.  An instance can be opened for each channel.
 
 **Usage**
-
-Before ``Catalogs`` can be used, you need to define at least one catalog or
-image server to be queried in ``ginga_config.py``. Here is an example for
-defining three cone search catalogs for guide stars::
-
-        def pre_gui_config(ginga):
-            from ginga.util.catalog import AstroPyCatalogServer
-
-            # Add Cone Search services
-            catalogs = [
-                ('The HST Guide Star Catalog, Version 1.2 (Lasker+ 1996) 1',
-                 'GSC_1.2'),
-                ('The PMM USNO-A1.0 Catalogue (Monet 1997) 1', 'USNO_A1'),
-                ('The USNO-A2.0 Catalogue (Monet+ 1998) 1', 'USNO_A2'),
-            ]
-            bank = ginga.get_ServerBank()
-            for longname, shortname in catalogs:
-                obj = AstroPyCatalogServer(
-                    ginga.logger, longname, shortname, '', shortname)
-                bank.addCatalogServer(obj)
-
-        def post_gui_config(ginga):
-            pass
-
-Then, start Ginga and then start the ``Catalogs`` local plugin from the
-channel you want to perform searchs on. You will see the catalogs listed
-in a drop-down menu on the plugin GUI.
-
 Draw a shape on the displayed image and adjust search parameters as desired.
 When you are ready, press on the button to perform the search.
 When search results are available, they will be displayed on the image and
 also listed in a table on the plugin GUI. You can click on either the table
 or the image to highlight selection.
 
+**Defining catalogs**
+You can customize the catalogs used by copying the `plugin_Catalogs.cfg`
+from the example configurations directory to your $HOME/.ginga directory and
+modifying the `catalog_sources` table defined inside.
+
+The format of this option is a list of dicts, where each dict defines a
+source and has the following fields:
+
+* shortname: str, the short name appearing in the control for selecting
+  a source in the plugin.  This is the name that will appear in the
+  drop-down menu in the catalogs control area of the GUI
+* fullname: str, the full name, should correspond *exactly* with the
+  name required by the `~astroquery.vo.conesearch` `catalog` parameter
```

Comment:
We don't have intersphinx set up for `astroquery`.
```suggestion
  name required by the ``astroquery.vo_conesearch`` ``catalog`` parameter
```